### PR TITLE
Multi-target Steeltoe against .NET 6/8

### PR DESCRIPTION
**Pull request by bart-vmware**
## Description

Adds multi-targeting against the RTM versions of .NET 6 and 8.

~Currently blocked by https://youtrack.jetbrains.com/issue/RSRP-494775/InspectCode-CleanupCode-hangs-on-.NET-8-RTM.~

## Quality checklist

- [x] Your code complies with our [Coding Style](https://github.com/SteeltoeOSS/.github/blob/main/contributing-docs/contributing-code-style.md).
- [x] You've updated unit and/or integration tests for your change, where applicable.
- [ ] You've updated documentation for your change, where applicable.
      If your change affects other repositories, such as [Documentation](https://github.com/SteeltoeOSS/Documentation), [Samples](https://github.com/SteeltoeOSS/Samples) and/or [MainSite](https://github.com/SteeltoeOSS/MainSite), add linked PRs here.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.
- [ ] You've added required license files and/or file headers (explaining where the code came from with proper attribution), where code is copied from StackOverflow, a blog, or OSS.

### CODE REVIEW

**Review by TimHess — DISMISSED** [review]
LGTM

File: shared.props [hunk 1]
```diff
@@ -19,6 +24,15 @@
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\Steeltoe.Release.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
```
[the remark this answers is not in the document]
Comment (bart-vmware) [reply shared.props]: Thanks, that's better indeed.

**Review by TimHess — APPROVED** [review]

**Review by TimHess — APPROVED** [review]
LGTM

**Pull request merged**